src/components/FindMovie/FindMovie.tsx

```diff
@@ -17,41 +82,58 @@ export const FindMovie: React.FC = () => {
               id="movie-title"
               placeholder="Enter a title to search"
               className="input is-dander"
+              value={query}
+              onChange={(event) => {
+                setQuery(event.target.value);
+                setIsVisible(false);
+              }}
             />
           </div>
 
-          <p className="help is-danger" data-cy="errorMessage">
-            Can&apos;t find a movie with such a title
-          </p>
+          {isVisible && (
+            <p className="help is-danger" data-cy="errorMessage">
+              Can&apos;t find a movie with such a title
```

Comment:
it's much easier to read string literals instead of charcodes
```suggestion
              {`Can't find a movie with such a title`}
```